hack/karmada-bootstrap.sh

```diff
@@ -124,7 +124,7 @@ export REGISTRY="swr.ap-southeast-1.myhuaweicloud.com/karmada"
 make images
 
 GO111MODULE=on go install "github.com/karmada-io/karmada/cmd/karmadactl"
-GOPATH=$(go env | grep GOPATH | awk -F '=' '{print $2}'| sed 's/\"//g')
```

Comment:
The GOPATH is a path list, we just need the first path.

According to `go help gopath`:
```
The GOPATH environment variable lists places to look for Go code.
On Unix, the value is a colon-separated string.
On Windows, the value is a semicolon-separated string.
On Plan 9, the value is a list.
```

Reply:
when i set `GOPATH=/root/go:/home/go`, and run `go env | grep GOPATH | awk -F '=' '{print $2}'| sed 's/\"//g'`, the output is `/root/go:/home/go`, we should find another way to split gopath list to select the first path

Reply:
go env GOPATH | awk -F ':' '{print $1}'

Reply:
that's it, considering different os, we can use `go env GOPATH | awk -F ':|;' '{print $1}'`, but i dont know On Plan 9, what is GOPATH like